### ISSUE TITLE
Make job properties mgmt features available to job developers

### ISSUE THREAD

**Issue by patmbolger** [AgileVentures/MetPlus_tracker#677]
![](https://dl.dropbox.com/s/a2kyoykoyftdksa/Screenshot%202018-10-31%2013.01.36.png?dl=0)

**Comment by patmbolger** [comment]
Enable job developers to access the menu "Admin > Job Properties"

**Comment by tansaku** [comment]
@patmbolger I realise you may not recall this one, but do we know why this was needed, and if it is still needed?

What does the "job properties mgmt feature" allow job developers to do that they couldn't previously?

**Comment by joaopapereira** [comment]
Yes, previously only the agency admin could change this.
If you login as a agency admin in the admin menu of the page you will see an option to change the properties.
The idea behind this story is to allow the job developer to also access it.

**Comment by tansaku** [comment]
@joaopapereira thanks 

What would be an example of what property of a job a job developer would want to change and why would they want to change it?

For example, would it be that the job address for applications is incorrect and the job developer would want to be able to change it to make sure it was correct?

Also, was this request from Chet? Or just an idea that the developers had?

**Comment by joaopapereira** [comment]
>What would be an example of what property of a job a job developer would want to change and why would they want to change it?

The main change we are talking here is the job skills and not 100% sure if the licenses also can change


> Also, was this request from Chet? Or just an idea that the developers had?

Request from Chet

**Comment by tansaku** [comment]
ah thanks @joaopapereira - so this is a situation where the job developer has been in touch with the employer and realises that the job requires some additional skills that should be added to the job properties?

The Job Developer role is someone who works for an agency helping match job seekers with jobs right?

**Comment by joaopapereira** [comment]
> ah thanks @joaopapereira - so this is a situation where the job developer has been in touch with the employer and realises that the job requires some additional skills that should be added to the job properties?

Not really. These properties are system wide and you can only set them in that admin spot. Let me exemplify with a workflow

As a Job Developer is creating a Job
*When* tries to add a skill that is not present in the platform
*Then* it access Admin > Job Properties
*And* add a new Skill to the Skill set
*Then* the Job Developer goes back to creating a Job
*And* the new Skill is displayed in the drop box

Same applied to edit job

**Comment by tansaku** [comment]
@joaopapereira thanks - so the motivation for making the change on the part of the stakeholder (the job developer), is that they are in the process of creating a job from scratch and they find they need an extra system-wide skill.  Presumably another approach would be to allow new skills to be added to the skill set from the job creation screen?

BTW is a Job Developer someone who entering a job on behalf of an employer?  A person who is working for the employer?  An employee of an agency?

**Comment by joaopapereira** [comment]
>@joaopapereira thanks - so the motivation for making the change on the part of the stakeholder (the job developer), is that they are in the process of creating a job from scratch and they find they need an extra system-wide skill. Presumably another approach would be to allow new skills to be added to the skill set from the job creation screen?

This was something that we talked about and the reason we decided to go with this was to make it easier to implement for now and then see is there were complaints about it

> BTW is a Job Developer someone who entering a job on behalf of an employer? A person who is working for the employer? An employee of an agency?

Job developer is an "employee" of an agency. I added the quotes here because it might not be an employee but someone that is helping the non profit.

**Comment by tansaku** [comment]
@joaopapereira thanks that's very helpful

**Comment by tansaku** [comment]
Hi @joaopapereira - looking at the system it seems like Job Developers cannot post jobs - it's only when we log in as company admins that we can post jobs.  There we can see the skills:

![](https://dl.dropbox.com/s/ferwdlv6p1ujtel/Screenshot%202018-10-31%2012.56.55.png?dl=0)

I can see that job developers can "edit" existing jobs, where they might be adding skills.  Is that what you mean?

Or is it that company admins need the ability to add more system-wide skills?

**Comment by joaopapereira** [comment]
@tansaku https://github.com/AgileVentures/MetPlus_tracker/issues/642 It was on the app but was removed because it had some issues

**Comment by tansaku** [comment]
@joaopapereira interesting - that makes it seem like we shouldn't push ahead on this feature without checking with Chet ...

**Comment by joaopapereira** [comment]
@tansaku I think it is a separate topic. But we can wait and talk to Chet about it
